cli/commands/cartridge.go

```diff
@@ -55,3 +65,14 @@ func setLogLevel() {
 		log.SetLevel(log.ErrorLevel)
 	}
 }
+
+func printVersion() {
+	versionString, err := version.BuildVersionString(projectPath, showRocksVersion)
+	fmt.Println(versionString)
+
+	if err != nil && showRocksVersion {
```

Comment:
The possible solution is to delegate it to `BuildVersionString` function. 

Let it be `PrintVersionString` function.

```go
if err := version.PrintVersionString(projectPath, projectPathIsSet, showRocksVersion); err != nll {
    return err
}
```

// PrintVersionString

```go
printCliVersion();
if err := printCartridgeVersion(); err != nil {
    if projectPathIsSet {
       return err
    }

    log.Warnf("Failed to show Cartridge version: %s", err)

    // possibly here we can return nil here, because getting rocks version will fail
}

if err := printRocksVersions(); err != nil {
    if showRocksVersion {
       return err
    }

   log.Warnf("Failed to show rocks versions: %s", err)

}
```